src/components/tables/tests/TableChildActionBar.spec.tsx

```diff
@@ -78,6 +86,21 @@ describe('<TableChildActionBar />', () => {
           });
         }).not.toThrow();
       });
+
+      it('should render without error if an actionBar prop, a filter prop, a datePicker prop and two predicate are passed', () => {
+        expect(() => {
+          mountComponentWithProps({
+            ...tablePropsMock,
+            actionBar: true,
+            filter: true,
+            predicates: [
+              { attributeName: 'email', attributeNameFormatter: _.identity, props: {} },
+              { attributeName: 'userName', attributeNameFormatter: _.identity, props: {} },
+            ],
+            datePicker: { datesSelectionBoxes: SELECTION_BOXES, attributeValue: 'date' },
```

Comment:
I like that...could you had this complex example in your example so we can try it?